### ISSUE TITLE
tensorflow:nightly-devel-gpu is broken

### ISSUE THREAD

**Issue by flx42** [tensorflow/tensorflow#4363]
Commit ["Fix nightly docker builds"](https://github.com/tensorflow/tensorflow/commit/414a5ddd47bba68661eaac08d5b6b7c92c38b808) broke the nightly builds :)
```
$ nvidia-docker run --rm -ti tensorflow/tensorflow:nightly-devel-gpu nvidia-smi
NVIDIA-SMI couldn't find libnvidia-ml.so library in your system. Please make sure that the NVIDIA Display Driver is properly installed and present in your system.
Please also try adding directory that contains libnvidia-ml.so to your system PATH.
```
You shouldn't override the existing value of `LD_LIBRARY_PATH`, we use it to point to the path where we mount the driver libraries mounted by nvidia-docker, see [here](https://github.com/NVIDIA/nvidia-docker/blob/master/ubuntu-14.04/cuda/7.5/runtime/Dockerfile#L38)

cc @3XX0 @caisq

**Comment by aselle** [comment]
@caisq could you please take a look at this. Perhaps set LD_LIBRARY_PATH to also include the previous LD_LIBRARY_PATH i.e.  maybe
```
ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:${LD_LIBRARY_PATH }
```

**Comment by flx42** [comment]
Was fixed by 60bb54e3111daf5a57008c97db05a91a93101fd7, closing.
